core/shared/src/main/scala/scalaz/zio/random/Random.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2017-2019 John A. De Goes and the ZIO Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package scalaz.zio.random
+
+import scalaz.zio.{ Chunk, UIO, ZIO }
+
+trait Random extends Serializable {
+  val random: Random.Service[Any]
+}
+object Random extends Serializable {
+  trait Service[R] extends Serializable {
+    val nextBoolean: ZIO[R, Nothing, Boolean]
+    def nextBytes(length: Int): ZIO[R, Nothing, Chunk[Byte]]
+    val nextDouble: ZIO[R, Nothing, Double]
+    val nextFloat: ZIO[R, Nothing, Float]
+    val nextGaussian: ZIO[R, Nothing, Double]
+    def nextInt(n: Int): ZIO[R, Nothing, Int]
+    val nextInt: ZIO[R, Nothing, Int]
+    val nextLong: ZIO[R, Nothing, Long]
+    val nextPrintableChar: ZIO[R, Nothing, Char]
+    def nextString(length: Int): ZIO[R, Nothing, String]
+  }
+  trait Live extends Random {
+    val random: Service[Any] = new Service[Any] {
+      import scala.util.{ Random => SRandom }
```

Comment:
Excuse my intrusion on this package given my absence in scala, but this is effectively just using a global random instance, even if it's observably pure. I'm not precisely sure that's a great thing to expose.